CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+forthcoming; for now, the `%ford` vane (in [`sys/vane/ford.hoon`][ford]) is
+some of the highest quality code in the kernel.
+
+[ford]: https://github.com/urbit/arvo/blob/master/sys/vane/ford.hoon
+
+## Kernel development
+
+Working on either C or non-kernel Hoon should not bring any surprises, but the
+Hoon kernel (anything under `sys/` in [urbit/arvo][arvo]) is bootstrapped from
+a so-called *pill*, and must be recompiled if any changes are made. This should
+happen automatically when you make changes, but if it doesn't, the command to
+manually recompile and install the new kernel is `|reset` in `dojo`.  This
+rebuilds from the `sys` directory in the `home` desk in `%clay`.
+
+Currently, `|reset` does not reload apps like `dojo` itself, which will still
+reference the old kernel. To force them to reload, make a trivial edit to their
+main source file (under the `app` directory) in `%clay`.
+
+[arvo]: https://github.com/urbit/arvo
+
+## The kernel and pills
+
+Urbit bootstraps itself using a binary blob called a pill (you can see it being
+fetched from `bootstrap.urbit.org` on boot).  This is the compiled version of
+the kernel (which you can find in the `sys` directory of [urbit/arvo][arvo]),
+along with a complete copy of the `urbit/arvo` repository as source.
+
+The procedure for creating a pill is often called "soliding." It is somewhat
+similar to `|reset`, but instead of replacing your running kernel, it writes
+the compiled kernel to a file. The command to solid is:
+
+```
+> .urbit/pill +solid
+```
+
+When the compilation finishes, your pill will be found in the
+`[pier]/.urb/put/` directory as `urbit.pill`.
+
+You can boot a new ship from your local pill with `-B`:
+
+```
+$ urbit -F zod -B path/to/urbit.pill my-fake-zod
+```
+
+Ordinarily, `https://bootstrap.urbit.org/latest.pill` will be updated
+to match whatever's on the HEAD of `master` in the `urbit/arvo` repository.
+Older pills are indexed by the first 10 characters of the `git` SHA1 of the
+relevant commit, i.e. as `git-[sha1].pill`.  The continuous integration build
+of the `urbit/arvo` repository uploads these pills for any successful build of
+a commit or pull request that affects the `sys/` directory.
+
+You can boot from one of these pills by passing the path to an Arvo working
+copy with `-A` (and `-s` for *search*):
+
+```
+$ git clone https://github.com/urbit/arvo
+$ urbit -F zod -A path/to/arvo -s my-fake-zod
+```
+
+## What to work on
+
+If you are not thinking of contributing with a specific goal in mind, the
+GitHub issue tracker is the first place you should look for ideas.  Issues are
+occasionally tagged with a priority and a difficulty; a good place to start is
+on a low-difficulty or low-priority issue.  Higher-priority issues are likely
+to be assigned to someone -- if this is the case, then contacting that person
+to coordinate before starting to work is probably a good idea.
+
+There is also a "help wanted" tag for things that we are especially eager to
```

Comment:
I've always interpreted "Help wanted" for issues/PRs in which someone's in-progress but stuck. Might be more valuable to mention "good contributor issue" here.

Reply:
Friendly reminder that you can also link to them, [like so](https://github.com/urbit/urbit/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+contributor+issue%22). (Turns out there's no open issues tagged as good for contributors. Maybe that needs to change!)